src/docdb/registerDocDBCommands.ts

```diff
@@ -27,8 +30,17 @@ export function registerDocDBCommands(actionHandler: AzureActionHandler, tree: A
     actionHandler.registerCommand('cosmosDB.createDocDBDocument', async (node?: IAzureParentNode) => {
         if (!node) {
             // #region Temporary changes to remove Documents node until viewing/editor stored procedures is implemented
-            // node = <IAzureParentNode>await tree.showNodePicker(DocDBDocumentsTreeItem.contextValue);
-            node = <IAzureParentNode>await tree.showNodePicker(DocDBCollectionTreeItem.contextValue);
+            node = <IAzureParentNode>await tree.showNodePicker(DocDBDocumentsTreeItem.contextValue);
+            //node = <IAzureParentNode>await tree.showNodePicker(DocDBCollectionTreeItem.contextValue);
+            // #endregion
+        }
+        await node.createChild();
+    });
+    actionHandler.registerCommand('cosmosDB.createDocDBStoredProcedure', async (node?: IAzureParentNode) => {
+        if (!node) {
+            // #region Temporary changes to remove Documents node until viewing/editor stored procedures is implemented
+            node = <IAzureParentNode>await tree.showNodePicker(DocDBDocumentsTreeItem.contextValue);
+            //node = <IAzureParentNode>await tree.showNodePicker(DocDBCollectionTreeItem.contextValue);
```

Comment:
I'll remove these comments in the next push. 

Same for those in DocDBCollectionTreeItem